agendas/2020-01-09.md

```diff
@@ -51,4 +52,5 @@ Example agenda item:
 1. Review agenda (2m, Lee)
 1. Review [previous meeting's action items](../notes/2019-12-05.md#action-items) (5m, Lee)
    - [Currently open action items](https://github.com/graphql/graphql-wg/issues?q=is%3Aissue+is%3Aopen+label%3A%22Action+item+%3Aclapper%3A%22)
+1. [Input Union RFC](https://github.com/graphql/graphql-spec/blob/master/rfcs/InputUnion.md)
```

Comment:
@binaryseed Just to keep a consistent format can you please add time estimation and yourself as the champion for this topic?